src/webgpu/api/validation/createSampler.spec.ts

```diff
@@ -0,0 +1,19 @@
+export const description = `
+createSampler validation tests.
+
+Test Plan: 
+
+* lodMinClamp:
+  * < 0
+  * = NAN
```

Comment:
We actually don't need to test NaN (or infinities)here because it's [invalid](https://heycam.github.io/webidl/#es-float) according to WebIDL. It would end up going in the idl/ tests later on.